Contest_API.md

```diff
@@ -2262,6 +2262,8 @@ By passing `group_id` with a valid group ID a scoreboard can be requested for th
 
 `contests/<id>/scoreboard?group_id=site1`
 
+Note that, when not passing a `group_id`, the teams indicated by `main_scoreboard_group` in the contest endpoint are used. 
```

Comment:
`_id` is missing from the property name.

My 2 cents - this still seems like we're waiting until the section on an optional parameter to describe what the primary behaviour is. I'd move this further up, e.g. in the main scoreboard description:
```
The scoreboard includes all teams indicated by `main_scoreboard_group_id` in the `contest` endpoint.
```

Reply:
Agreed. Fixed.